### DISCUSSION TITLE
TALKBACK

### DISCUSSION THREAD

**Discussion by ReeceyDRAGON** [community/community#39559] [Accessibility] [answered]
May someone please further explain in some detail how TALKBACK accessibility is used and for what it can be used to achieve results wise??🙏

**Comment by jfullstackdev** [comment]
the talkback that you might be referring to is Google Talkback 

https://github.com/google/talkback

you can ask question there by raising an issue

**Comment by alexandervauer** [comment, marked as answer]
In short terms it's there to help people with eye sight problems, so that they easier can navigate through your digital solution. It is often used to read out loud what kind of text is presented on the screen, and what kind of function the buttons you use have. Therefore you often have to double-tap components on the screen, to first listen to it's functionality, and then execute it on the second.

**Comment by cehfisher** [comment]
Hi @ReeceyDRAGON - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/39559#discussioncomment-4227620). Could you please close this issue? Thank you! (cc @queenofcorgis)
